### ISSUE TITLE
HeatmapGenerator and PafGenerator run so slowly

### ISSUE THREAD

**Issue by zllrunning** [donnyyou/torchcv#4]
In methods/pose/open_pose.py, when batch size equels to 32, 

```
heatmap = self.heatmap_generator(data_dict['kpts'], input_size, maskmap=maskmap)
vecmap = self.paf_generator(data_dict['kpts'], input_size, maskmap=maskmap)
```

These two lines will take about 40 seconds every iteration. It is too time consuming. The network runs only 0.03 second. 

Are there any details to reduce runing time I ignored ?

**Comment by youansheng** [comment]
Thanks for your issue! You could do this in dataloader with many workers. I will solve it as soon as possible.

**Comment by youansheng** [comment]
Or you could rewrite it with Cython or Cuda C.

**Comment by zllrunning** [comment]
Tks you for your immediate reply.  I have already used 8 workers in dataloader, so this might not be feasible. Maybe rewriting it with Cython or Cuda C can solve this problem.

**Comment by zllrunning** [comment]
There is still a problem.
In loss/modules/pose_modules.py, 
line 28 
```
loss += weights[i] * self.mse_loss(inputs[i]*mask, targets)
```
```
inputs[i].size() : torch.Size([32, 19, 46, 46])
targets.size() : torch.Size([32, 19, 46, 46])
mask.size() : torch.Size([32, 46, 46])
```
 **mask = mask.unsqueeze(1)**　 is needed in my experiment.

**Comment by youdonny** [comment]
This bug has been fixed. Now the GT map generators are so fast!!!
